arkane/multidimensionalTorsionsTest.py

```diff
@@ -0,0 +1,109 @@
+#                                                                             #
+# Permission is hereby granted, free of charge, to any person obtaining a     #
+# copy of this software and associated documentation files (the 'Software'),  #
+# to deal in the Software without restriction, including without limitation   #
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,    #
+# and/or sell copies of the Software, and to permit persons to whom the       #
+# Software is furnished to do so, subject to the following conditions:        #
+#                                                                             #
+# The above copyright notice and this permission notice shall be included in  #
+# all copies or substantial portions of the Software.                         #
+#                                                                             #
+# THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR  #
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,    #
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER      #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING     #
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER         #
+# DEALINGS IN THE SOFTWARE.                                                   #
+#                                                                             #
+###############################################################################
+
+"""
+This script contains unit tests of the :mod:`arkane.multidimensionalTorsions` module.
+"""
+
+import unittest
+import os
+import zipfile
+
+from arkane.multidimensionalTorsions import HinderedRotor2D, HinderedRotorClassicalND
+from arkane.statmech import determine_qm_software
+
+class TestHinderedRotor2D(unittest.TestCase):
+    """
+    Contains unit tests of the StatmechJob class.
+    """
+    @classmethod
+    def setUp(cls):
+        """A method that is run before each unit test in this class"""
+        path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                                         'data', 'CH2CHOOH', 'CH2CHOOHscans')
+        if not os.path.exists(path):
+            zippath = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                                             'data', 'CH2CHOOH', 'CH2CHOOHscans.zip')
+            with zipfile.ZipFile(zippath,'r') as zip_ref:
+                zip_ref.extractall(os.path.split(path)[0])
+        
+        cls.hd2d= HinderedRotor2D(calcPath=path,name="r0",torsigma1=1,
+                      torsigma2=1,symmetry='b',pivots1=[6,7],pivots2=[1,6],top1=[7,8],top2=[6,7,8])
+              
+    def test_q2dtor_setup(self):
+        if os.path.isfile(self.hd2d.q2dtor_path):
```

Comment:
I think you can use `unittests.skipIf` instead.

Reply:
done